modules/remotecontrol/Receiver.js

```diff
@@ -30,7 +33,11 @@ const logger = getLogger(__filename);
  *
  * @type {Transport}
  */
-const transport = getJitsiMeetTransport();
+const transport = new Transport({
+    backend: new PostMessageTransportBackend({
+        postisOptions: { scope: 'jitsi-remote-control' }
```

Comment:
You could explain to me why this scope option is needed? I'm not familiar with the bugs present without scope.